### PR TITLE
Open details pages in a popup modal

### PR DESCRIPTION
**Pull request by Ramadan1011**
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by jamshidbek7375** [comment]
You have removed the edit.
Don't we need it?

**Review by jamshidbek7375 — COMMENTED** [review]
How do we edit checks now?

**Comment by jamshidbek7375** [comment]
> > How do we edit checks now? Before exam image
> 
> we don't edit reviews

ok

**Review by jamshidbek7375 — APPROVED** [review]

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicHandover.cshtml [hunk 1]
```diff
@@ -2,61 +2,111 @@
...
+            </div>
+            <div class="col text-center">
+                1 oylik yoqilg'i me'yori
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center" style="border-bottom: 0.1px solid #d9d9d9;">
+                @Model.OneYearMeduimFuelConsumption
+            </div>
+            <div class="col text-center" style="border-bottom: 0.1px solid #d9d9d9;">
```
Comment (FirdavsAX) [inline CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicHandover.cshtml:99]: There why you use inline css ,  You can replace css to site.css file or create css file in root folder .

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicAcceptance.cshtml [hunk 2]
```diff
@@ -2,61 +2,111 @@
...
+        <div class="row content">
+            <div class="col text-center">
+                1 yillik yoqilg'i me'yori
+            </div>
+            <div class="col text-center">
+                1 oylik yoqilg'i me'yori
+            </div>
+        </div>
+        <div class="row value">
+            <div class="col text-center" style="border-bottom: 0.1px solid #d9d9d9;">
```
Comment (FirdavsAX) [inline CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicAcceptance.cshtml:96]: You can use css styles in site.css or create any .css file in root folder ?

**Review by SharifovDeveloper — APPROVED** [review]

File: CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicHandover.cshtml (hunk 1 above)
Comment (Boburm1rzo) [reply CheckDrive.Web/CheckDrive.Web/Views/Shared/_CarDetailsForMechanicHandover.cshtml:99]: I agree with you

**Review by diyorzz — APPROVED** [review]

**Pull request closed without merging**